awxkit/awxkit/api/pages/inventory.py

```diff
@@ -353,7 +354,7 @@ def remove_group(self, group):
 
 class Groups(page.PageList, Group):
 
-    pass
+    NATURAL_KEY = ('group',)
```

Comment:
I'm pretty sure 'group' isn't a field or related on the Group model.  You should check the Django models under the awx/ subtree of this repo to get a sense of what fields are used for the uniqueness constraint for this model.

Reply:
Also, I just noticed that this is on Groups.  The plural page type shouldn't get the natural key, only the singular version.